app/models/metric/ci_mixin/capture.rb

```diff
@@ -17,6 +17,20 @@ def queue_name_for_metrics_collection
     ems.metrics_collector_queue_name
   end
 
+  def split_capture_intervals(start_time, end_time)
+    intervals = []
+    start_hour = start_time
+    while start_hour != end_time
```

Comment:
Yeah, this should definitely be a `<`.  Or just use step_value as mentioned above.